Formula/tezos-accuser-011-PtHangz2.rb

```diff
@@ -34,6 +34,9 @@ class << self
   def make_deps
```

Comment:
Oh I saw now only that you also bumped the release number.

Since you changed formulae as well, I think that we should bump the `version` here too (and make new bottles)

Reply:
Right, forgot about it